static/pages/volunteer/Volunteer.js

```diff
@@ -59,4 +63,25 @@ oppiaFoundationWebsite.controller('VolunteerPage', ['$scope', '$document',
         throw Error('No such element');
       }
     };
+    $scope.submitContactUsForm = function(
+        fullName, email, comment, event) {
```

Comment:
'event' is a keyword in some old browsers, let's change it to evt.

Reply:
Done!